package.json

```diff
@@ -43,6 +43,7 @@
     "dirty-chai": "^2.0.1",
     "gulp": "^3.9.1",
     "libp2p-mplex": "~0.7.0",
+    "libp2p-pnet": "github:libp2p/js-libp2p-pnet#feat/initial",
```

Comment:
Can this be released?

Reply:
Yes it can, I am just waiting for a thumbs up on https://github.com/libp2p/js-libp2p-pnet/pull/1